parameters/hardware/actuators/spindle_controller.ops

```diff
@@ -82,12 +82,12 @@ SPINDLE_Safety.Nm						= N
 SPINDLE_Safety.maxJointErrors			= MAXERR
 SPINDLE_Safety.motorSaturations			= array(100.0)
 SPINDLE_Safety.maxConSatTime			= 1.0
-SPINDLE_Safety.additional_safeties		= strings("endswitch")
+#SPINDLE_Safety.additional_safeties		= strings("endswitch")
```

Comment:
@MaxBaeten What is the status of the endswitch?

Reply:
Should be fixed. Replace cabling or sensor. It is on Henk's hardware list. We should definitely not continue with this hack in place.

Reply:
So merge this PR and create issue? Or revert this hack, so we are reminded that the sensor is not working?